modules/ref_quay-integration-config-fields.adoc

```diff
@@ -0,0 +1,27 @@
+:_content-type: REFERENCE
+[[quay-integration-config-fields]]
+
+== QuayIntegration configuration fields
+
+The following configuration fields are available for the QuayIntegration custom resource:
+
+[cols="4a,2a,2a",options="header"]
+|===
+|Name |Description |Schema
+|allowlistNamespaces +
+(Optional) | A list of namespaces to include. |Array
+|clusterID +
+(Required) |The ID associated with this cluster. |String
+|credentialsSecret.key +
+(Required) | The secret containing credentials to communicate with the Quay registry. |Object
+|denylistNamespaces +
+(Optional) | A list of namespaces to exclude. |Array
+|insecureRegistry +
+(Optiona) |Whether to skip TLS verification to the Quay registry |Boolean
+|organizationPrefix +
+(Optional) | The prefix assigned to organizations. |String
```

Comment:
this configuration is not used, have filed a bug for it: https://issues.redhat.com/browse/PROJQUAY-4157